sources/academy/tutorials/apify_scrapers/cheerio_scraper.md

```diff
@@ -70,9 +70,13 @@ And as we already know, there's only one.
 
 ```js
 // Using Cheerio.
-return {
-    title: $('header h1').text(),
-};
+async function pageFunction(context) {
+    const { $ } = context;
+    // ... rest of your code can come here
+    return {
+        title: $('header h1').text(),
```

Comment:
not that important, but we could use line highlighting for those examples that are not containing additional boilerplate

https://docusaurus.io/docs/markdown-features/code-blocks#line-highlighting

Reply:
How would you apply it here?

Reply:
the example is about the cheerio usage, so i would highlight those lines maybe?